winerp/client.py

```diff
@@ -240,7 +260,7 @@ async def ping(self, client=None, timeout: int = 60) -> bool:
         resp = await self.__get_response(_uuid, asyncio.get_event_loop(), timeout=timeout)
         return resp.get("success", False)
 
-    async def call_function(self, destination, object_identifier, func_name, *args, **kwargs) -> bool:
+    async def _call_function(self, destination, object_identifier, func_name, *args, **kwargs) -> bool:
```

Comment:
It will result in error as it has not been updated in the responseObject in payload.py

Reply:
i ll check

Reply:
https://github.com/nouman0103/winerp/blob/46d16a8fbeb23429fa5248dc33f778f4d9a0865c/winerp/lib/payload.py#L169